tests/slowproxy_test.go

```diff
@@ -179,4 +246,10 @@ func TestSlowProxy(t *testing.T) {
 	// Test shuttling data in both directions, with close initiaited once by each side
 	runTest(true)
 	runTest(false)
+
+	require.NoError(t, sp.Close())
+	spErr := <-spErrCh
+	require.Error(t, spErr)
+	require.Contains(t, spErr.Error(), "use of closed network connection")
```

Comment:
Could this use `errors.Is()` to be more robust against message changes?

Reply:
Assuming that you are looking for `io.ErrClosedPipe`

Reply:
Unfortunately the error is a `net.OpError`, and you have to look at the message to see that it is the use of closed network connection error. I verified this with a debugger. This is also test code for a test helper class.

Reply:
Is the net.OpError wrapping a [net.ErrClosed](https://pkg.go.dev/net#pkg-variables) so you could use `IsError(err, net.ErrClosed)` as they describe in the link?  Not to make work, not a blocker in any sense.